vdr/types/interface.go

```diff
@@ -45,7 +45,7 @@ type DocCreator interface {
 	// The ID in the provided DID document will be ignored and a new one will be generated.
```

Comment:
Not part of this PR, but still; `The ID in the provided DID document will be ignored and a new one will be generated.`?

Reply:
Since you can't set your own DID